src/Nethermind/Nethermind.AuRa.Test/Reward/AuRaRewardCalculatorTests.cs

```diff
@@ -83,6 +90,18 @@ public void constructor_throws_ArgumentNullException_on_null_transactionProcesso
             Action action = () => new AuRaRewardCalculator(_auraParameters, _abiEncoder, null);
             action.Should().Throw<ArgumentNullException>();
         }
+
+        [Test]
+        public void constructor_throws_ArgumentException_on_BlockRewardContractTransition_higher_than_BlockRewardContractTransitions()
+        {
+            _auraParameters.BlockRewardContractTransitions = new Dictionary<long, Address>()
+            {
+                {2, Address.FromNumber(2)}
```

Comment:
does the address 2 mean something here? if not then worth using TestItem.AddressA

Reply:
Address 2 is in the precompiles space